docs/notes/instrumentation_measurement/geth.md

```diff
@@ -82,6 +82,10 @@ What is going on between `CaptureState`s (every opcode):
 - (moved after `execute` in current fork) `in.cfg.Tracer.CaptureState(...)` tracing itself, just before `execute` of an opcode
 - `logged = true` if tracing is on
 
+### Execution environment
```

Comment:
let's have a quick doc on how to run the measurements (ideally using the `program_generator.py` and `measurements.py` scripts) using the docker container, for both envs, i.e. the `docker run` commands needed.

We might also consider mounting a volume to easily move data to/from the container, but that can be version 2

Reply:
FYI @lukasz-glen, for `geth` this works for me, didn't try the `openethereum` one:

```

sudo docker run \
  --rm --privileged --security-opt seccomp:unconfined \
  -it measurements-geth \
  bash -c "cd src && python3 program_generator/program_generator.py generate --fullCsv | python3 instrumentation_measurement/measurements.py measure --sampleSize=5 --nSamples=1"

```